dev/config/keycloak/import/ltt-realm.json

```
@@ -208,7 +208,10 @@
          "oauth2.device.authorization.grant.enabled": "false",
          "client.secret.creation.time": "1703195110",
          "backchannel.logout.session.required": "true",
-         "backchannel.logout.revoke.offline.tokens": "false"
+         "backchannel.logout.revoke.offline.tokens": "false",
+         "backchannel.logout.url": "https://ubu.mcjustin.dev.cirg.uw.edu/auth/oidc/logoutCallback",
```

@ivan-c  I wasn't sure what to do about this... perhaps we want to define a variable for it? I'm not sure how to do that...

---

@mcjustin 

You'll need to reuse, or define a new env var for that URL. Daniel's implemented [similar templating](https://github.com/uwcirg/ltt-environments/pull/27/files#diff-20b1317eda463480f19c5c19bebf3379d29195092fe98a65566fe0f18b31f242R157) in #27.

In addition to the suggested changes, you'll also need to add something like the below to the `environment` section of `keycloak` in docker-compose.yaml. I created a new env var below, but other ways of templating/combining those strings may make more sense
```
diff --git a/dev/docker-compose.yaml b/dev/docker-compose.yaml
index 4120103..9165ac3 100644
--- a/dev/docker-compose.yaml
+++ b/dev/docker-compose.yaml
@@ -153,6 +153,8 @@ services:
 
       KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: admin
+     
+      KEYCLOAK_CPRO_BACKCHANNEL_LOGOUT_URL: https://${BASE_DOMAIN}/auth/oidc/logoutCallback
```

```suggestion
         "backchannel.logout.url": "${KEYCLOAK_CPRO_BACKCHANNEL_LOGOUT_URL}",
```

---

@ivan-c  does this okay?
https://github.com/uwcirg/ltt-environments/pull/28/files#diff-fc93d9014ebae50dfe3db8c3db1e5637bd66ad9633cc3548a9380aa3f16dad99R212

---

@mcjustin 

$BASE_DOMAIN is set in .env, so it's only for templating in docker-compose.yaml. I pushed a commit (4629220) that sets up a new environment variable for Keycloak based on $BASE_URL.